tsl/src/continuous_aggs/create.c

```diff
@@ -477,59 +477,77 @@ cagg_add_trigger_hypertable(Oid relid, int32 hypertable_id)
 	ts_cache_release(hcache);
 }
 
-/* add additional indexes to materialization table for the columns derived from
- * the group-by column list of the partial select query
- * if partial select query has:
- * GROUP BY timebucket_expr, <grpcol1, grpcol2, grpcol3 ...>
- * index on mattable is <grpcol1, timebucketcol>, <grpcol2, timebucketcol> ... and so on.
- * i.e. #indexes =(  #grp-cols - 1)
+/*
+ * Add additional indexes to materialization hypertable for the columns derived from
+ * the group-by column list of the partial select query.
+ *
+ * If partial select query has:
+ *   GROUP BY timebucket_expr, <grpcol1, grpcol2, grpcol3 ...>
+ * Index on mattable is <grpcol1, grpcol2, grpcol3> ... and so on.
```

Comment:
Shouldnt this be <grpcol1, grpcol2, grpcol3,time_bucket_expr>

Reply:
Fixed